src/main/java/service/Arithmetic/Arithmetic.java

```diff
@@ -0,0 +1,6 @@
+package service.Arithmetic;
```

Comment:
패키지 이름은 관례에 따라 소문자로만 작성해주시면 좋을 것 같습니다.
```suggestion
package service.arithmetic;
```